test/e2e/framework/secretproviderclass/secretproviderclass.go

```diff
@@ -0,0 +1,61 @@
+// +build e2e
+
+package secretproviderclass
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/secrets-store-csi-driver-provider-azure/test/e2e/framework"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/secrets-store-csi-driver/apis/v1alpha1"
+)
+
+// CreateInput is the input for Create.
+type CreateInput struct {
+	Creator   framework.Creator
+	Config    *framework.Config
+	Name      string
+	Namespace string
+	Spec      v1alpha1.SecretProviderClassSpec
```

Comment:
Might be good to create a json template for the spec to catch regressions for all the expected values in parameters. WDYT?

Reply:
Based on offline discussion:
1. We'll keep the [examples](https://github.com/Azure/secrets-store-csi-driver-provider-azure/tree/master/examples) dir which will contain sample manifests for each supported configuration.
2. For schema validation, we'll need to ensure with thorough unit testing that the AKV provider handles all expected values.